File: ITManagementServer/src/main/java/models/requestModels/requests/UpdateRequestRequestModel.java
package models.requestModels.requests;

public class UpdateRequestRequestModel {
    private int RequestId;
    private String ResolveNote;
    private int RequestStatus;

    public int getRequestId() {
        return RequestId;
    }

    public void setRequestId(int requestId) {
        RequestId = requestId;
    }

    public String getResolveNote() {
        return ResolveNote;
    }

    public void setResolveNote(String resolveNote) {
        ResolveNote = resolveNote;
    }

    public int getRequestStatus() {
        return RequestStatus;
    }

    public void setRequestStatus(int requestStatus) {
        RequestStatus = requestStatus;
    }
}

File: ITManagementClient/Models/RequestModels/Requests/UpdateRequestRequestModel.cs
using ITManagementClient.Models.Enums;

namespace ITManagementClient.Models.RequestModels.Requests
{
    public class UpdateRequestRequestModel
    {
        public int RequestId { get; set; }

        public string ResolveNote { get; set; }

        public RequestStatus RequestStatus { get; set; }
    }
}